static/src/js/components/GranuleResults/GranuleResultsItem.scss

```diff
@@ -24,6 +28,14 @@
     font-weight: 400;
     line-height: 1.4em;
     word-wrap: break-word;
+
+    &.selected {
+      color: #3498db;
```

Comment:
We should be using variables for these colors. I think you want `$color__brand--dark` here. Same for below.